x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1048,4 +1048,14 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.host_isolation.allowed',
+    first_supported_version: '8.6',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.host_isolation.allowed',
+      {
+        defaultMessage: 'Message TBD. Default: true',
```

Comment:
```
A value of 'false' disallows host isolation activity on Linux endpoints, regardless of whether host isolation is supported. Note that if a host is currently not isolated, it will refuse to isolate, and likewise, a host will refuse to release if it is currently isolated. A value of 'true' will allow Linux endpoints to isolate if supported. Default: 'true'
```

@nfritts @roxana-gheorghe for viz